src/ProductConstructionService/ProductConstructionService.FeedCleaner/FeedCleanerJob.cs

```diff
@@ -84,7 +83,15 @@ await Parallel.ForEachAsync(
                     try
                     {
                         cancellationToken.ThrowIfCancellationRequested();
-                        await feedCleaner.CleanFeedAsync(feed);
+
+                        if (FeedConstants.MaestroManagedSymbolFeedNamePattern.IsMatch(feed.Name))
+                        {
+                            await feedCleaner.CleanSymbolFeedAsync(managedFeeds, feed);
```

Comment:
I think we should be doing this a bit different: split the feeds into two lists, one for package and one for symbol feeds, then we go through the package ones, and then we go through the symbol ones. I mean it's not a big issue, but in this case, some symbol feeds won't get cleaned up until next run

Reply:
Fair point, will change

Reply:
Done